MTM-60219 Exports component documentation (#2476)
SoftwareAG/c8y-docs: content/cockpit/exports-bundle/working-with-exports.md

---
weight: 20

> title: Adding exports to widgets

Author: @MWindel I would go with more generic name, like:
```suggestion
title: Exports integration with Data Points based features
```
I have no idea how to shorten this name, but the point is export feature in not bound only to widgets.

layout: bundle
outputs:
  - html
  - json
sector:
  - app_enablement
---

> While the export functionality itself is a standalone feature, you can combine it with other features that allow you to select data points. To use the export functionality in this way, you must integrate it with the respective feature, for example, the [Data points table](/cockpit/widgets-collection/#data-point-table) widget.

Reviewer: ```suggestion
While the export functionality itself is a standalone feature, you can combine it with other features that allow you to select data points. To use the export functionality in this way, you must integrate it in the respective feature, for example, the [Data points table](/cockpit/widgets-collection/#data-point-table) widget.
```

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

- To generate exports in a widget: READ permission permission type "Measurement"
{{< /c8y-admon-req >}}

### To create an export in a widget {#to-create-an-export-in-a-widget}

> The following example shows you how to integrate the export functionality with the [Data point table](/cockpit/widgets-collection/#data-point-table) widget.

Author: @MWindel I would change it to:
```suggestion
The following example shows you how the export functionality is integrated with the [Data point table](/cockpit/widgets-collection/#data-point-table) widget.
```
